server/src/main/java/org/elasticsearch/TransportVersions.java

```diff
@@ -184,6 +184,7 @@ static TransportVersion def(int id) {
     public static final TransportVersion INCLUDE_INDEX_MODE_IN_GET_DATA_STREAM = def(9_023_0_00);
     public static final TransportVersion MAX_OPERATION_SIZE_REJECTIONS_ADDED = def(9_024_0_00);
     public static final TransportVersion RETRY_ILM_ASYNC_ACTION_REQUIRE_ERROR = def(9_025_0_00);
+    public static final TransportVersion ESQL_SERIALIZE_BLOCK_TYPE_CODE = def(9_026_0_00);
```

Comment:
I noticed this pir has 8.19 label as well. Was it intentional? This introduces 9x transport version

Reply:
Yes, I will backport this to 8.19